README.md

```diff
@@ -45,8 +45,10 @@ video in both the tabs!
 
 ## Examples
 
-The project contains some common use-case examples for the Twilio Video JS SDK.
+The project contains some common use-case examples for the Twilio Video JS SDK. After running the application
+by following the instructions above, click on these links to run the examples.
```

Comment:
I think this indicates we ought to find a way to include these links in the app, e.g. a navigation bar or something?

Reply:
@markandrus I'll add an `examples/index.html` page linking to all the examples.